opslevel/datasource_opslevel_services.go

```diff
@@ -49,7 +49,12 @@ func datasourceServicesRead(d *schema.ResourceData, client *opslevel.Client) err
 	case "product":
 		services, err = client.ListServicesWithProduct(value, nil)
 	case "tag":
-		services, err = client.ListServicesWithTag(opslevel.NewTagArgs(value), nil)
+		var tagArgs opslevel.TagArgs
+		tagArgs, err = opslevel.NewTagArgs(value)
```

Comment:
`var` declaration is necessary here because otherwise linter complaisn about ineffectual assignment to `err` on line 57, because `err` is defined before the switch.